### PR TITLE
Review sharp examples

### PR DESCRIPTION
**Pull request by voferreira**
# Description of the problem

- Part of the example reupdating efforts, this PR is to fix small stuff here and there I spotted while testing the sharp examples.

# Description of the solution

- Small fixes here and there.
- Additionally, I suggest changing internally `sharp-immersed-boundary-solver` to `sharp-immersed-boundary`

# How Has This Been Tested?

- All examples have been tested.
- All tests passed as well.

# Future changes

- Other sections may need to be updated. So far only incompressible and sharp have been tested.

### CODE REVIEW

File: doc/source/examples/sharp-immersed-boundary/sedimentation-1-particle/sedimentation-1-particle.rst [hunk 1]
```diff
@@ -42,7 +42,7 @@ All the container walls have no-slip boundary conditions except at the top of th
 Parameter File
 ---------------
 
-We explain every part of this prm file in detail. In each section of the parameter file, we describe relevant parameters. The omitted parameters are only user preference parameters and do not impact the simulation results. For more detail on these parameter we suggest visiting the :doc:`../../../parameters/parameters`.
+We explain every part of this `prm` file in detail. In each section of the parameter file, we describe relevant parameters. The omitted parameters are only user preference parameters and do not impact the simulation results. For more detail, we suggest visiting the :doc:`../../../parameters/parameters`.
```
Comment (blaisb) [inline doc/source/examples/sharp-immersed-boundary/sedimentation-1-particle/sedimentation-1-particle.rst:45]:
```suggestion
We explain every part of this parameter file in detail. In each section of the parameter file, we describe relevant parameters. The omitted parameters are only user preference parameters and do not impact the simulation results. For more detail, we suggest visiting the :doc:`../../../parameters/parameters`.
```

File: doc/source/examples/sharp-immersed-boundary/sedimentation-1-particle/sedimentation-1-particle.rst [hunk 2]
```diff
@@ -55,7 +55,7 @@ Simulation and IO Control
       set time end           = 1.3    # End time of simulation
       set output name        = out    # Prefix for VTU outputs
       set output frequency   = 1      # Frequency of simulation output
-      set subdivision        = 1      # Mesh subdivision when outputend
+      set subdivision        = 1      # Mesh subdivision when output end
```
Comment (blaisb) [inline doc/source/examples/sharp-immersed-boundary/sedimentation-1-particle/sedimentation-1-particle.rst:58]: Subdivision=1 does not do anything, it does not subdivide at all. Consequently, better remove this line man...

File: doc/source/examples/sharp-immersed-boundary/sedimentation-1-particle/sedimentation-1-particle.rst [hunk 3]
```diff
@@ -81,8 +81,8 @@ Physical Properties
 * The ``kinematic viscosity`` is set to  0.6041666666666. This value is derived from the case description by dividing :math:`\mu_f` by :math:`\rho_f`.
 
 .. note:: text
-	The fluid density is not set here as the sharp interface solver has his own definition of the density of the fluid. This will be modified soon to unify both definitions in the parameter. 
-	
+    The fluid density is not set here as the sharp interface solver has its own definition of the density of the fluid. This will be modified soon to unify both definitions in the parameter.
```
Comment (blaisb) [inline doc/source/examples/sharp-immersed-boundary/sedimentation-1-particle/sedimentation-1-particle.rst:84]: Is this still an issue? if so, clearly you should put in the project of things to fix

File: doc/source/examples/sharp-immersed-boundary/sphere-carreau-with-sharp-interface/sphere-carreau-with-sharp-interface.rst [hunk 4]
```diff
@@ -106,7 +106,7 @@ We define the boundary conditions in order to have an inlet velocity of :math:`1
     end
 
 .. note::
-	Since using a `deal.ii mesh <https://www.dealii.org/current/doxygen/deal.II/namespaceGridGenerator.html>`_, the boundary ``id = 1`` is by default the second boundary in the *x* axis, hence the free boundary in this problem. This is why all boundary indices are shifted.
+    Since using a `deal.ii mesh <https://www.dealii.org/current/doxygen/deal.II/namespaceGridGenerator.html>`_, the boundary ``id = 1`` is by default the second boundary in the *x* axis, hence the free boundary in this problem. This is why all boundary indices are shifted.
```
Comment (blaisb) [inline doc/source/examples/sharp-immersed-boundary/sphere-carreau-with-sharp-interface/sphere-carreau-with-sharp-interface.rst:109]: Is this really necessary?

File: doc/source/examples/sharp-immersed-boundary/sphere-carreau-with-sharp-interface/sphere-carreau-with-sharp-interface.rst [hunk 5]
```diff
@@ -243,44 +243,44 @@ Results
...
 -----------------------------	
 
 * **High-order methods** : Lethe supports higher order interpolation. This can yield much better results with an equal number of degrees of freedom than traditional second-order (Q1-Q1) methods, especially at higher Reynolds numbers.
 * **Reynolds number** : By changing the inlet velocity, it can be interesting to see the impact of the shear-thinning behavior on the effective drag force.
-* **Non-Newtonian parameters** : It can also be interesting to change the Carreau model parameters, i.e. changing the slope to appreciate the change in behavior.
+* **Non-Newtonian parameters** : It can also be interesting to change the Carreau model parameters, i.e. changing the slope to appreciate the behavior change.
 
 .. note::
-	It is not possible to use the Carreau model for a shear-thickening flow. You should change the model to power-law, and use the decribed parameters in :doc:`../../../parameters/cfd/physical_properties`
+    It is not possible to use the Carreau model for a shear-thickening flow. You should change the model to power-law, and use the decribed parameters in :doc:`../../../parameters/cfd/physical_properties`
```
Comment (blaisb) [inline doc/source/examples/sharp-immersed-boundary/sphere-carreau-with-sharp-interface/sphere-carreau-with-sharp-interface.rst:286]: Is this necessary? it's a limitation of the model itself, not the numerics. It really is not made for that. Even Pierre Carreau says it, so it has to be true.

File: examples/sharp-immersed-boundary/sedimentation-1-particle/sedimentation-1-particle.prm [hunk 6]
```diff
@@ -138,7 +138,7 @@ subsection mesh adaptation
   # Fraction of refined elements
   set fraction refinement = 0.05
 
-  # How the fraction of refinement/coarsening are interepretedChoices are
+  # How the fraction of refinement/coarsening are interepreted. Choices are
```
Comment (blaisb) [inline examples/sharp-immersed-boundary/sedimentation-1-particle/sedimentation-1-particle.prm:141]:
```suggestion
  # How the fraction of refinement/coarsening are interpreted. Choices are
```

**Review by blaisb — APPROVED** [review]
Very few things, great work Vic

File: doc/source/examples/sharp-immersed-boundary/sedimentation-1-particle/sedimentation-1-particle.rst (hunk 3 above)
Comment (voferreira) [reply doc/source/examples/sharp-immersed-boundary/sedimentation-1-particle/sedimentation-1-particle.rst:84]: It is no longer an issue. Sorry for letting this one pass. Nice catch.

File: doc/source/examples/sharp-immersed-boundary/sphere-carreau-with-sharp-interface/sphere-carreau-with-sharp-interface.rst (hunk 4 above)
Comment (voferreira) [reply doc/source/examples/sharp-immersed-boundary/sphere-carreau-with-sharp-interface/sphere-carreau-with-sharp-interface.rst:109]: I think we should keep it. It is not necessary but I think someone skipping straight to this example would be wondering why.

File: doc/source/examples/sharp-immersed-boundary/sphere-carreau-with-sharp-interface/sphere-carreau-with-sharp-interface.rst (hunk 5 above)
Comment (voferreira) [reply doc/source/examples/sharp-immersed-boundary/sphere-carreau-with-sharp-interface/sphere-carreau-with-sharp-interface.rst:286]: Not necessary, but I think Pierre Carreau himself would appreciate having us remind people not to do what he says is not meant to be done :upside_down_face:

**Comment by voferreira** [comment]
@blaisb The two comments missing on Carreau are on your call. Let me know your final decision.

File: doc/source/examples/sharp-immersed-boundary/sedimentation-1-particle/sedimentation-1-particle.rst [hunk 7]
```diff
@@ -260,7 +256,7 @@ Linear Solver
 
 * The ``relative residual`` is set to 1e-4. This is small enough, so we don't under-resolve our matrix and do extra non-linear steps because of it, and at the same, it doesn't require too many ``gmres`` iterations.
 
-* The ``ilu preconditioner fill`` is set to 0. This means that we have a Jacobi preconditioner. This is the cheapest option. In this case, we are able to use this option without having to do too many ``gmres`` iterations. It requires less computational time to do a few more  ``gmres`` iterations than building the preconditioner and doing fewer ``gmres`` iterations.
+* The ``ilu preconditioner fill`` is set to 0. This means that we have a Jacobi preconditioner. This is the cheapest option. In this case, we can use this option without having to do too many ``gmres`` iterations. It requires less computational time to do a few more  ``gmres`` iterations than building the preconditioner and doing fewer ``gmres`` iterations.
```
Comment (Luckabarbeau) [inline doc/source/examples/sharp-immersed-boundary/sedimentation-1-particle/sedimentation-1-particle.rst:259]:
```suggestion
* The ``ilu preconditioner fill`` is set to 0. This is the cheapest option. In this case, we can use this option without having to do too many ``gmres`` iterations. It requires less computational time to do a few more  ``gmres`` iterations than building the preconditioner and doing fewer ``gmres`` iterations.
```

**Review by Luckabarbeau — APPROVED** [review]
Nice job! Very good maintenance work :)

File: doc/source/examples/sharp-immersed-boundary/sphere-carreau-with-sharp-interface/sphere-carreau-with-sharp-interface.rst (hunk 5 above)
Comment (blaisb) [reply doc/source/examples/sharp-immersed-boundary/sphere-carreau-with-sharp-interface/sphere-carreau-with-sharp-interface.rst]: it's chill to keep it like this.

**Pull request merged**